_topic_map.yml

```diff
@@ -158,13 +158,13 @@ Topics:
   File: creating-project-other-user
   Distros: openshift-enterprise,openshift-origin
 ---
-Name: Administering clusters
-Dir: administering_clusters
+Name: Administration
```

Comment:
@adellape - there is no Administration bucket in our current list. Have you talked to @aheslin about this?

Reply:
There is also no "Administering clusters". We have discussed the need for better directories, but nothing's final. I had another PR that I needed to put a topic into this directory, so I renamed here for the meantime.

But, I am going to switch this yet again to "Masters" to match the spreadsheet, and hopefully we'll get a better name soon, and then change it again to the final name.